kensu/numpy/__init__.py

```diff
@@ -417,9 +417,18 @@ def wrapper(*args, **kwargs):
     array = array(np.array)
 
     def extract_sc(kensu, nd):
-        orig_ds = eventually_report_in_mem(kensu.extractors.extract_data_source(nd, kensu.default_physical_location_ref,
+        try:
+            orig_ds = eventually_report_in_mem(kensu.extractors.extract_data_source(nd, kensu.default_physical_location_ref,
                                                                                 logical_naming=kensu.logical_naming))
-        orig_sc = eventually_report_in_mem(kensu.extractors.extract_schema(orig_ds, nd))
+            orig_sc = eventually_report_in_mem(kensu.extractors.extract_schema(orig_ds, nd))
+        except Exception:  # thrown by default extract_data_source/extract_schema
+            msg = "Kensu numpy.array got an unexpected argument which can not be fully tracked (yet): " + str(type(nd))
+            logging.warning(msg)
+            print(msg)
```

Comment:
print?

Reply:
pytest was hiding my logs, finally found a way to fight it `--log-cli-level=INFO` :)